components/Layout.js

```diff
@@ -64,6 +65,7 @@ export default function Layout(props) {
           onSignOut: () => {
             signOut({ callbackUrl: process.env.AUTH_ECAS_GLOBAL_LOGOUT_URL })
           },
+          customLink: Link,
```

Comment:
This doesn't seem to do anything, maybe have a conversation with the DS team to either get an idea on how we should be using this prop or to get them to adjust their component

Reply:
Fixed.